games-fps/prboom-plus/prboom_plus-2.5.1.4.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Doom source port"
```

Comment:
Shall we change the summary to something more descriptive like this: "Enhanced clone of the classic first-person shooter Doom"?